src/conn/conn_api.c

```diff
@@ -1855,6 +1863,45 @@ __wt_verbose_config(WT_SESSION_IMPL *session, const char *cfg[])
 }
 
 /*
+ * __conn_verbose_cmd --
+ *	Run the verbose commands.
+ */
+static int
+__conn_verbose_cmd(WT_SESSION_IMPL *session, const char *config)
+{
+	static const struct verbose_cmd_struct {
+		const char *name;			/* command name */
+		int (*func)(WT_SESSION_IMPL *);		/* command */
+	} verbose_cmd[] = {
+		 { "dump_cache", __wt_verbose_dump_cache },
+		 {   "dump_txn", __wt_verbose_dump_txn },
```

Comment:
Some whitespace issue going on there.  I think this function needs a different name and/or a big comment.  Maybe call it `conn_verbose_once_cmd` to make it obvious it is only for one-time diagnostic.  Or `conn_verbose_diagnostics`.